src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -99,13 +98,7 @@ const StreamGroup = createReactClass({
         )}
         <GroupSummary w={[8 / 12, 8 / 12, 6 / 12]} ml={canSelect ? 1 : 2} mr={1} flex="1">
           <EventOrGroupHeader data={data} query={query} />
-          <EventOrGroupExtraDetails
-            group
-            {...data}
-            groupId={id}
-            orgId={orgId}
-            projectId={projectId}
-          />
+          <EventOrGroupExtraDetails group {...data} />
```

Comment:
Does `data` have the groupId, orgId, and projectId keys? Or were those props unused?

Reply:
I've refactored the component to not use these anymore but always read these from params. The reason for this is  to conditionally show different links when params.projectId is defined